### PR TITLE
Move all CSS constants into a separate file

### LINKED ISSUE #49: Move all CSS constants into a separate file
#### Description
SCSS allows for you to write [variables](https://sass-lang.com/documentation/variables), and then you can [import](https://sass-lang.com/documentation/at-rules/import) the file to use everywhere else.

We want to be able to just import things so the code looks cleaner and we would be able to change colors easily if we want to.

#### Files
You can probably put this in `App.scss` where `App.js` is, I don't have a huge preference

#### Tasks
- [ ] Create scss file for constants

- [ ] Find colors in other current SCSS files (and any others if you can find them in the design) and add them as SCSS variables

- [ ] Import the file in other scss files and replace hardcoded values with variables

### PR DESCRIPTION
**Pull request by nroy20**
## Description :star2:

Removed all color constants in scss files and replaced them with variables from a new variable file.
Fixes #49

### Sample from Navigation.scss:
image
### Sample from _app.scss (variable file):
image
 
## TODOs :star:
- [ ] Review

### CODE REVIEW

File: backend/api/views/appointment.py [hunk 1]
```diff
@@ -72,3 +72,16 @@ def put_appointment(id):
     appointment.save()
 
     return create_response(status=200, message=f"Success")
+
+
+# DELETE request for appointment by appointment id
```
Comment (AngelaLuo49021) [inline backend/api/views/appointment.py:77]: lol why is this here

**Review by AngelaLuo49021 — DISMISSED** [review]
Overall good stuff but update this branch on what's on master currently and see if anything else needs to be changed.

**Review by chyku — DISMISSED** [review]
looks fine, did u branch off your delete appointments branch LOL

File: frontend/src/components/css/Appointments.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import "_app";
```
Comment (chyku) [inline frontend/src/components/css/Appointments.scss:1]: kinda crazy that you can just import the file this smoothly

File: frontend/package.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "^4.2.2",
```
Comment (chyku) [inline frontend/package.json:6]: nice nice

**Review by chyku — APPROVED** [review]
lgtm

**Review by AngelaLuo49021 — APPROVED** [review]
good stuff
